manifests/params.pp

```diff
@@ -225,4 +225,7 @@
       $plugin_prefix = 'smart_proxy_'
     }
   }
+
+  $puppet_use_cache = undef
+  $puppet_cache_location = '/var/lib/smart-proxy/cache'
```

Comment:
:+1:  for ```/var/cache/foreman-proxy```. IMHO we should just create this as empty directory belonging to the foreman-proxy user in the RPM and DEB packages.